src/state/apis/nft/parsers/alchemy.ts

```diff
@@ -159,13 +159,20 @@ export const parseAlchemyNftToNftItem = async (
 
   const medias = maybeMedias.unwrap()
 
+  const maybeBalance: Pick<NftItem, 'balance'> = (
+    alchemyNft.tokenType.toLowerCase() === ASSET_NAMESPACE.erc1155
+      ? { balance: bnOrZero((alchemyNft as OwnedNft).balance).toNumber() }
+      : {}
+  ) as Pick<NftItem, 'balance'>
```

Comment:
Any reason this isn't the below?

```ts
const maybeBalance: Pick<NftItem, 'balance'> =
    alchemyNft.tokenType.toLowerCase() === ASSET_NAMESPACE.erc1155
      ? { balance: bnOrZero((alchemyNft as OwnedNft).balance).toString() }
      : {}
```

Reply:
Disregard me I'm derp - bd8cac55a2fdc5cfbcfef844bcdd62ce7b20d0c2